### ISSUE TITLE
More documentation needed, please

### ISSUE THREAD

**Issue by David263** [black7375/Firefox-UI-Fix#51]
**Describe the bug**
Please describe how to get the "compact" menus, with smaller vertical spacing.
**Expected behavior**
I expected the menus to be fixed to look like before release 89.0. They are still too spread out vertically. The toolbars look better, though.
**Screenshots**

**Environment (please complete the following information):**
 - Firefox Version: [ 	89.0]
 - OS:
   - [ ] Linux
   - [10 Home ] Windows
   - [ ] Mac
   - [ ] Others: 
 - Theme:
   - [x ] Light
   - [ ] Dark
   - [ ] Alpenglow(System: Light)
   - [ ] Alpenglow(System: Dark)
   - [ ] System Default
     - GTK Theme Name(Only Linux User): [sample] Adwaita, Adwaita-Dark, Breath, Breath-Dark

**Additional context**
Also, there is no background color difference between the three toolbars, hard to see. Also, the active tab is not well highlighted--my eyes are not that good.

**Comment by black7375** [comment]
1. Do you mean context menu? or hamburger menu(Panel)?

2. How about using [photon-style](https://github.com/black7375/Firefox-UI-Fix/tree/photon-style)?
It's easier to distinguish because there's a tab line.
![image](https://user-images.githubusercontent.com/25581533/120728828-3dda7980-c4cd-11eb-8154-3df8459d735c.png)

**Comment by David263** [comment]
No, I don't mean the context menu or hamburger menu. I don't know how to use photon-style because there are no instructions.

Besides, I have had to uninstall Firefox-UI-Fix because of the serious Pinned Tabs bug. I don't plan to try this fix again until it is really working well. It has been released way too early.

**Comment by black7375** [comment]
Likewise, you can download it from the link above.

This is a beta program, not even 1.0.
There are plans to remove the pinned tab close button issue after that.
